src/TouristInfoCards.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div>
+      <img src={props.cityImg} alt="..." />
+      <div>
+        <h5>{props.city}</h5>
+        <p>{props.cityInfo}</p>
+        <a href="peoplemakeglasgow.com">Go somewhere</a>
```

Comment:
All of your tourist info cards will link to the Glasgow website. How could we fix that?